lib/utils/ajvErrorMessageBuilder.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
While such practice has no precedence in this repo. I would put this util aside of `ConfigSchemaHandler` class (as it serves just it's purpose), here it confuses by suggesting it's a general util to be used in different disconnected places, while it's not the case.

Imo good way, would be to create folder `lib/ConfigSchemaHandler` and within it put class at `index.js` and this util at `ajvErrorMessageBuilder.js`

Reply:
I could move this file to `lib/configSchema` folder which currently contains only one `index.js` file with the initial schema.

Does this make sense?

Reply:
I think it'll be better if it's in `lib/classes/ConfigSchemaHandler` (sorry I missed `classes` token in first message). So it's by the class that relies on it.

I would keep `lib/configSchema` strictly for plain JSON schema (to not mix concerns), and if there's no point to split core schema into multiple files I would just make it `lib/configSchema.js` (not a folder)

Reply:
Thanks for the suggestion. I'll rename it this way